icalevents/icalparser.py

```diff
@@ -267,6 +274,9 @@ def parse_events(content, start=None, end=None, default_span=timedelta(days=7)):
 
     # Keep track of the timezones defined in the calendar
     timezones = {}
+    if 'X-WR-TIMEZONE' in calendar:
+        timezones[str(calendar['X-WR-TIMEZONE'])] = gettz(str(calendar['X-WR-TIMEZONE']))
```

Comment:
Should we maybe add a comment explaining what `X-WR-TIMEZONE` is? 
Especially since it's an extension and not quite standard.